nautobot/core/settings.yaml

```diff
@@ -0,0 +1,1655 @@
+    "details": |-
+      !!! warning
+          If enabling indefinite changelog retention, it is recommended to periodically delete old entries.
+          Otherwise, the database may eventually exceed capacity.
+    "environment_variable": "NAUTOBOT_CHANGELOG_RETENTION"
+    "is_constance_config": true
+    "type": "integer"
+  "CONFIG_CONTEXT_DYNAMIC_GROUPS_ENABLED":
+    "default": false
+    "description": >-
+      If `True`, it will be possible to apply Config Context objects to Devices and Virtual Machines
+      via Dynamic Group membership. When set to `False` this behavior will not be available.
+    "details": |-
+      !!! warning
+          With a large number of dynamic groups, enabling this could invoke a performance penalty when
+          processing Config Contexts.
+    "environment_variable": "NAUTOBOT_CONFIG_CONTEXT_DYNAMIC_GROUPS_ENABLED"
+    "type": "boolean"
+  "CONTENT_TYPE_CACHE_TIMEOUT":
+    "default": 0
+    "description": >-
+      The number of seconds to cache the content type accessible via a object's class property
+      `Object._content_type_cached`. This can save frequent calls to `ContentType.objects.get_for_model(model)`.
+      Set this to `0` to disable caching.
+    "environment_variable": "NAUTOBOT_CONTENT_TYPE_CACHE_TIMEOUT"
+    "type": "integer"
+    "version_added": "1.6.0"
+  "CORS_ALLOWED_ORIGINS":
+    "default": []
+    "description": "A list of origins that are authorized to make cross-site HTTP requests."
+    "details": |-
+      An Origin is defined by [the CORS RFC Section 3.2](https://tools.ietf.org/html/rfc6454#section-3.2) as a
+      URI `scheme + hostname + port`, or one of the special values `'null'` or `'file://'`.
+      Default ports (HTTPS = 443, HTTP = 80) are optional here.
+
+      The special value `null` is sent by the browser in
+      ["privacy-sensitive contexts"](https://tools.ietf.org/html/rfc6454#section-6), such as when the client
+      is running from a `file://` domain.
+      The special value `file://` is sent accidentally by some versions of Chrome on Android as per this bug.
+
+      Example:
+
+      ```python
+      CORS_ALLOWED_ORIGINS = [
+          'https://example.com',
+          'https://sub.example.com',
+          'http://localhost:8080',
+          'http://127.0.0.1:9000'
+      ]
+      ```
+    "items":
+      "format": "uri"
+      "type": "string"
+    "type": "array"
+  "CORS_ALLOWED_ORIGIN_REGEXES":
+    "default": []
+    "description": >-
+      A list of strings representing regexes that match Origins that are authorized to make cross-site HTTP requests.
+    "details": |-
+      Useful when [`CORS_ALLOWED_ORIGINS`](#cors_allowed_origins) is impractical, such as when you have a
+      large number of subdomains.
+
+      Example:
+
+      ```python
+      CORS_ALLOWED_ORIGIN_REGEXES = [r"^https://\w+\.example\.com$"]
+      ```
+    "items":
+      "$ref": "#/definitions/regex"
+    "type": "array"
+  "CORS_ALLOW_ALL_ORIGINS":
```

Comment:
out of order?

Reply:
On that note, should our for loops in the jinja template be sorting these alphabetically to account for cases like this?